packages/content/package.json

```diff
@@ -4,8 +4,8 @@
   "description": "Module to interact with ArcGIS Hub Content in Node.js and modern browsers.",
   "main": "dist/node/index.js",
   "unpkg": "dist/umd/content.umd.js",
-  "module": "dist/esm/index.js",
-  "es2017": "dist/es2017/index.js",
+  "module": "dist/es5/index.js",
+  "es2017": "dist/esm/index.js",
```

Comment:
Ha! I think this might explain something that was unexpected (to me at least) in the built files.

Reply:
yeah, we're still defaulting ("module") to ESM b/c I didn't want to do a breaking change yet.